packages/xod-client-electron/test-func/ide.spec.js

```diff
@@ -0,0 +1,140 @@
+import fsp from 'fsp';
+import path from 'path';
+
+import { assert } from 'chai';
+
+import prepareSuite from './prepare';
+
+import { TRIGGER_MAIN_MENU_ITEM } from '../src/testUtils/triggerMainMenu';
+
+const workspacePath = subPath => path.resolve(__dirname, '../../../workspace/', subPath);
+
+describe('IDE', () => {
```

Comment:
Maybe better call the test case something like `blink.spec.js`. I think it better reflects what’s going on here.